### PR TITLE
update the way the outfile names are computed

### PR DESCRIPTION
**Pull request by rmarabini**
When executing different cryosparc protocols I get error messages similar to:

#00037:   Unable to execute the copy: Files or directory does not exist:  [Errno 2] No such file or directory: '/home/roberto/cryoSPARC/scipion_projects/2022_06_20_mx2369_pkv-carmen/P3/J188/cryosparc_P3_J188_001754_particles.cs'

That is, cryosparc has finished and produced the output but scipion cannot find them. After closer inspection it seems that the name of the output filename is wrong. That is, scipion searches for "cryosparc_P3_J188_001754_particles.cs" but the actual name is "cryosparc_P3_J188_01754_particles.cs" (there was an extra zero).

I have changed the code so lines as

        csParticlesName = "cryosparc_%s_%s_00%s_particles.cs" % (
become
        csParticlesName = "cryosparc_%s_%s_%05d_particles.cs" % (

This solved my problem but I do not know if it will be OK in all cases. 

In summary: under some conditions the actual code produces output file names that are wrong. This pull request fixes the problem for my particular case but a more systematic check is recommended.

### CODE REVIEW

**Comment by fonsecareyna82** [comment]
Hi @rmarabini, one question: did you update cryosparc to version 4.0.0?

**Comment by fonsecareyna82** [comment]
Sorry @rmarabini ...I see that the problem is not an update. Yes, in your case you will solve but it will not be general. That is an issue that I assigned myself a while ago and I have not resolved.

**Comment by rmarabini** [comment]
> Sorry @rmarabini ...I see that the problem is not an update. Yes, in your case you will solve but it will not be general. That is an issue that I assigned myself a while ago and I have not resolved.

Hi @fonsecareyna82  Do you think that changing 

 csParticlesName = "cryosparc_%s_%s_00%s_particles.cs" % (

by 

 import glob
csParticlesName = glob.glob("/path_to_file/cryosparc_%s_%s_*%s_particles.cs" % (...))

will do the trick?
